internal/gcs/guestconnection.go

```diff
@@ -190,6 +190,25 @@ func (gc *GuestConnection) DeleteContainerState(ctx context.Context, cid string)
 	return gc.brdg.RPC(ctx, rpcDeleteContainerState, &req, &resp, false)
 }
 
+func (gc *GuestConnection) UpdateContainer(ctx context.Context, cid string, resources interface{}) (err error) {
+	ctx, span := trace.StartSpan(ctx, "gcs::GuestConnection::UpdateContainer")
```

Comment:
I think for new code the plan was to stop with the :: convention in our logs and use periods but my mind is hazy.

Reply:
@kevpar can you comment on this? I wanted to be consistent with the other guest connection traces